about.html

```diff
@@ -0,0 +1,194 @@
+                    <p>
+                        If your field of practive evolve around, the cloud, Big Data, IoT, Metaverse, Quantum Computing, SDN and any other gallery of next gneneration Technologies which we haven't mentioned, do well to be present with us this year 2022 and share in the deep knowledge of the cloud and beyound.
+
+                    </p>
+                </div>
+                <div class="headline-paragraph-contact">
+                    <p>
+                        Please contact us per Email for any further questions about PAN African Cloud Computing Summit 2022!
+                    </p>
+                    <a href="https://www.google.com">PANACCS2022@PANNigeria.org</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container">
+        <h2 class="main-title">
+            PAN African Cloud Computing  Global Summit 2022 Logo
+        </h2>
+        <p class="main-paragraph">
+            The logo of PANACCS Summit 2022 was decided through the logo competition from 4th August to 7th September.
+        </p>
+        <p class="main-paragraph-desktop">
+            The logo of PANACCS Summit 2022 was decided through the logo competition from 4th August to 7th September.
+            online public voting and Selection Committee's judging process, the logo from Balandele Juwon won the prize.
+            After consultation with Mr. Nalandele we optimized the prize-winning-logo for this year's summit as following:
+        </p>
+        <div class="main-logo">
+            <img src="./Image/PAN_Logo.jpg" alt=" PANACCS Logo">
+        </div>
+    </main>
+    <section class="container past-events">
+        <h2 class="past-events-title">
+            See the past PANACCS Summits
+        </h2>
+        <div class="past-events-description">
+            <p>
+                Take a look at the two PANACCS Summits which took place in Cairo and also Lome.
+            </p>
+        </div>
+        <div class="past-event-cards">
+            <div class="past-event-card-1">
+                <h2 class="past-event-date">
+                    2020
+                </h2>
+                <p class="past-event-site">
+                    PANACCS Summit 2020 in Cairo
+                </p>
+            </div>
+            <div class="past-event-card-2">
+                <h2 class="past-event-date">
+                    2018
+                </h2>
+                <p class="past-event-site">
+                    PANACCS Summit 2018 in Lome
+                </p>
+            </div>
+        </div>
+    </section>
+    <footer>
+        <div class="partners">
+            <div class="partner-title">
+                <p>Partner</p>
+            </div>
+            <div class="container partner-group">
+                <p>Amazon</p>
+                <p>Google</p>
+                <p>Nvidia</p>
+                <p>CircleCI</p>
+                <p>airbnb</p>
+            </div>
+        </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you use a `ul` tag as a container instead of a `div` tag for your partners' list and for each partner item a `<li>` tag instead of a `<p>` tag for best semantics tags.
Also, for the **Partner** title a `hx(e.g.: h2, h3,...)` tag will be better than a  `p` tag.